src/main/resources/liquibase.xml

```diff
@@ -33,4 +33,16 @@
             </column>
         </createTable>
     </changeSet>
+    <changeSet id="202009140505" author="Neeraj/Aditi">
```

Comment:
I think given we have not deployed this changelog anywhere yet, we could change the existing changelog of createTable.

Reply:
Done